unumpy/_multimethods.py

```diff
@@ -827,29 +827,29 @@ def broadcast_to(array, shape, subok=False):
 
 def _normalize_axis(ndim, axis, allow_repeats=False):
     if isinstance(axis, numbers.Number):
```

Comment:
```suggestion
    if not isinstance(axis, collections.abc.Sequence):
```

Reply:
I think this makes a nice and simple check for nesting:

```suggestion
    if not isinstance(axis, collections.abc.Sequence):
        axis = operator.index(axis)
        axis_pos = ndim + axis if axis < 0 else axis
```

The `operator.index` statement fails with the same error as NumPy:

```python
In [2]: ones = onp.ones(shape=(1, 2, 1, 3))                                     

In [3]: onp.squeeze(ones, axis=(0, (2,)))                                       
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-a467e7557a93> in <module>
----> 1 onp.squeeze(ones, axis=(0, (2,)))

<__array_function__ internals> in squeeze(*args, **kwargs)

~/miniconda3/envs/unumpy/lib/python3.7/site-packages/numpy/core/fromnumeric.py in squeeze(a, axis)
   1481         return squeeze()
   1482     else:
-> 1483         return squeeze(axis=axis)
   1484 
   1485 

TypeError: 'tuple' object cannot be interpreted as an integer
```

The same statement is also useful for the `Axis is out of bounds` error message.